src/components/Editor/index.js

```diff
@@ -372,7 +381,8 @@ class Editor extends PureComponent {
     }
 
     if (gutter !== "CodeMirror-foldgutter") {
-      toggleBreakpoint(line + 1);
+      const sourceId = selectedSource ? selectedSource.get("id") : "";
```

Comment:
Lets move the `selectedSource` higher up. 

```js
if (! selectedSource) {
return 
}
```

we can do the same thing in `onToggleBreakpoint`